lib/scrabble.rb

```diff
@@ -1,13 +1,53 @@
 class Scrabble
-  def point_values
-    {
+  attr_reader     :point_values,
+                  :counter
+
+  def initialize
```

Comment:
You could have kept this as just the `point_vaues` method and used the method's return value as the storage for the values hash.